server/test/services/geo/esri/EsriClientTest.java

```diff
@@ -0,0 +1,78 @@
+package services.geo.esri;
+
+import static org.junit.Assert.*;
+import static play.mvc.Results.*;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableList;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import java.io.IOException;
+import java.util.*;
+import java.util.concurrent.CompletionStage;
+import org.junit.*;
+import play.libs.Json;
+import play.libs.ws.*;
+import play.routing.RoutingDsl;
+import play.server.Server;
+import services.Address;
+import services.geo.AddressSuggestion;
+import services.geo.AddressSuggestionGroup;
+
+public class EsriClientTest {
+  private EsriClient client;
+  private WSClient ws;
+  private Server server;
+  private Config config;
+
+  @Before
+  public void setup() {
+    config = ConfigFactory.load();
+    server =
```

Comment:
**suggestion (non-blocking):** Add a comment that this is setting up fake data from a test file for the service.

Reply:
added comment on setting up of mack data